src/jobs/sync_job.ts

```diff
@@ -31,7 +31,13 @@ export abstract class SyncJob {
 
     async start(): Promise<boolean> {
         JobLog.setToRunning(this._jobLog);
-        const result = await this._doTheJob();
+        let result
+        try {
+            result = await this._doTheJob();
+        } catch (ex: any) {
+            console.log(ex)
```

Comment:
Tady by se asi měl taky uložit job log...

Reply:
Honza vyřeší job log. Případné Sentry bude pak na nás.

Reply:
joblog added